bib/cmd/bootc-image-builder/image.go

```diff
@@ -154,6 +156,195 @@ func pipelinesForDiskImage(c *ManifestConfig, rng *rand.Rand) (image.ImageKind,
+			"rpcbind",
+			"selinux-policy-targeted",
+			"systemd",
+			"tar",
+			"xfsprogs",
+			"xz",
+		},
+	}
+
+	img.ExtraBasePackages = img.ExtraBasePackages.Append(rpmmd.PackageSet{
+		Include: []string{
+			"aajohan-comfortaa-fonts",
+			"abattis-cantarell-fonts",
+			"alsa-firmware",
+			"alsa-tools-firmware",
+			"anaconda",
+			"anaconda-dracut",
+			"anaconda-install-env-deps",
+			"anaconda-widgets",
+			"atheros-firmware",
+			"audit",
+			"bind-utils",
+			"bitmap-fangsongti-fonts",
+			"brcmfmac-firmware",
+			"bzip2",
+			"cryptsetup",
+			"curl",
+			"dbus-x11",
+			"dejavu-sans-fonts",
+			"dejavu-sans-mono-fonts",
+			"device-mapper-persistent-data",
+			"dmidecode",
+			"dnf",
+			"dracut-config-generic",
+			"dracut-network",
+			"efibootmgr",
+			"ethtool",
+			"fcoe-utils",
+			"ftp",
+			"gdb-gdbserver",
+			"gdisk",
+			"glibc-all-langpacks",
+			"gnome-kiosk",
+			"google-noto-sans-cjk-ttc-fonts",
+			"grub2-tools",
+			"grub2-tools-extra",
+			"grub2-tools-minimal",
+			"grubby",
+			"gsettings-desktop-schemas",
+			"hdparm",
+			"hexedit",
+			"hostname",
+			"initscripts",
+			"ipmitool",
+			"iwlwifi-dvm-firmware",
+			"iwlwifi-mvm-firmware",
+			"jomolhari-fonts",
+			"kbd",
+			"kbd-misc",
+			"kdump-anaconda-addon",
+			"kernel",
+			"khmeros-base-fonts",
+			"less",
+			"libblockdev-lvm-dbus",
+			"libibverbs",
+			"libreport-plugin-bugzilla",
+			"libreport-plugin-reportuploader",
+			"librsvg2",
+			"linux-firmware",
+			"lldpad",
+			"lsof",
+			"madan-fonts",
+			"mtr",
+			"mt-st",
+			"net-tools",
+			"nfs-utils",
+			"nmap-ncat",
+			"nm-connection-editor",
+			"nss-tools",
+			"openssh-clients",
+			"openssh-server",
+			"ostree",
+			"pciutils",
+			"perl-interpreter",
+			"pigz",
+			"plymouth",
+			"python3-pyatspi",
+			"rdma-core",
+			"realtek-firmware",
+			"rit-meera-new-fonts",
+			"rng-tools",
+			"rpcbind",
+			"rpm-ostree",
+			"rsync",
+			"rsyslog",
+			"selinux-policy-targeted",
+			"sg3_utils",
+			"sil-abyssinica-fonts",
+			"sil-padauk-fonts",
+			"smartmontools",
+			"spice-vdagent",
+			"strace",
+			"systemd",
+			"tar",
+			"tigervnc-server-minimal",
+			"tigervnc-server-module",
+			"udisks2",
+			"udisks2-iscsi",
+			"usbutils",
+			"vim-minimal",
+			"volume_key",
+			"wget",
+			"xfsdump",
+			"xfsprogs",
+			"xorg-x11-drivers",
+			"xorg-x11-fonts-misc",
+			"xorg-x11-server-Xorg",
+			"xorg-x11-xauth",
+			"xrdb",
+			"xz",
+		},
+	})
+	img.ISOLabelTempl = "Container-Installer-%s"
+
+	var customizations *blueprint.Customizations
+	if c.Config != nil && c.Config.Blueprint != nil {
+		customizations = c.Config.Blueprint.Customizations
+	}
+
+	img.Users = users.UsersFromBP(customizations.GetUsers())
+	img.Groups = users.GroupsFromBP(customizations.GetGroups())
+
+	switch c.Architecture {
+	case arch.ARCH_X86_64:
+		img.Platform = &platform.X86{
+			BasePlatform: platform.BasePlatform{
+				ImageFormat: platform.FORMAT_ISO,
+			},
+			BIOS:       true,
+			UEFIVendor: "fedora",
```

Comment:
(I still think this hardcoding of UEFI vendor in Go code is problematic...it gets fixed when using bootupd for images that use that; this relates to the above)

Reply:
> I still think this hardcoding of UEFI vendor in Go code is problematic

It is, but until we have a robust way of detecting the distro of the container, we need to define it somewhere (or ask the user for it).
And since bootupd will solve this, should we be thinking of other solutions?

Reply:
Wait, will bootupd solve this also for this ISO? I don't expect bootupd to be used there.

Reply:
Uh, yeah, sorry, got a bit confused there.  This is the bootloader for the ISO itself and I imagine we'll keep doing grub install for that and will need to set the vendor (based on the distro).

Reply:
Since we're building the ISO from hardcoded repos anyway, let's keep this hardcoded for now as well.  Maybe add a comment that it will change once we get multi-distro support.